eng/Npm.Workspace.nodeproj

```diff
@@ -22,7 +22,11 @@
 
   <Target Name="Restore" DependsOnTargets="_VerifyNPMCache">
     <Message Text="Restoring NPM packages..." Importance="high" />
-    <Exec Command="npm ci --offline" WorkingDirectory="$(MSBuildThisFileDirectory).." />
+    <PropertyGroup>
```

Comment:
Don't we need this in `SourceBuild.props` instead? https://github.com/dotnet/aspnetcore/blob/b61a93689c4ac61b571904c0e0e665914083c71a/eng/SourceBuild.props#L110-L112

Reply:
Yes, you are right